vindbaarheid/vindbaarheid.rdfs.ttl

```diff
@@ -0,0 +1,472 @@
+    rdfs:domain schema:Thing ;
+    owl:equivalentProperty dct:title ;
+    rdfs:label "name"@en ;
+    rdfs:comment "The name of the item."@en .
+
+schema:partOfEpisode a rdf:Property ;
+    # rdfs:domain premis:IntellectualEntity ;
+    rdfs:domain schema:CreativeWork ; #? Clip?
+    rdfs:range schema:Episode ;
+    rdfs:label "part of episode"@en ;
+    rdfs:comment "The episode to which this clip belongs."@en . 
+
+schema:partOfSeries a rdf:Property ;
+    # rdfs:domain premis:IntellectualEntity ;
+    rdfs:domain schema:CreativeWork ;
+    rdfs:range schema:CreativeWorkSeries ;
+    rdfs:label "part of series"@en ;
+    rdfs:comment "The series to which this episode or season belongs."@en .
+
+schema:publisher a rdf:Property ;
+    # rdfs:domain premis:IntellectualEntity, schema:PerformanceRole ;
+    rdfs:domain schema:CreativeWork ; #? Ook Role als domain?
+    rdfs:range owl:unionOf (schema:Person org:Organization schema:Role) ;
+    owl:equivalentProperty dct:publisher ;
+    rdfs:label "publisher"@en ;
+    rdfs:comment "The publisher of the creative work."@en .
+
+schema:roleName a rdf:Property ;
+    rdfs:domain schema:PerformanceRole ;
+    rdfs:range skos:Concept ;
+    rdfs:label "role name"@en ;
+    rdfs:comment "A role played, performed or filled by a person or organization."@en .
+
+schema:seasonNumber a rdf:Property ;
+    rdfs:domain schema:CreativeWorkSeason ;
+    rdfs:label "season number"@en ;
+    rdfs:comment "Position of the season within an ordered group of seasons."@en .
+
+schema:spatialCoverage a rdf:Property ;
+    # rdfs:domain premis:IntellectualEntity ;
+    rdfs:domain schema:CreativeWork ;
+    rdfs:range schema:Place ;
+    owl:equivalentProperty dct:spatial ;
+    rdfs:label "spatial coverage"@en ;
+    rdfs:comment "The spatialCoverage of a CreativeWork indicates the place(s) which are the focus of the content."@en .
+
+schema:temporalCoverage a rdf:Property ;
+    # rdfs:domain premis:IntellectualEntity ;
+    rdfs:domain schema:CreativeWork ;
+    rdfs:label "temporal coverage"@en ;
+    owl:equivalentProperty dct:temporal ;
+    rdfs:comment "The temporalCoverage of a CreativeWork indicates the period that the content applies to, i.e. that it describes, either as a DateTime or as a textual string indicating a time period in ISO 8601 time interval format."@en .
+
+schema:transcript a rdf:Property ;
+    # rdfs:domain meemoo:DigitalRepresentation ;
+    rdfs:domain schema:CreativeWork ;
+    rdfs:label "transcript"@en ;
+    rdfs:comment "If this MediaObject is an AudioObject or VideoObject, the transcript of that object."@en .
+
+#? Added props
+#? Nieuwe relaties meenemen of enkel kortste links tussen gevonden entities?
+#? Origine van metadata: https://pav-ontology.github.io/pav/ (createdWith, createdBy, importedFrom)
+#? Role? Waarom stap daartussen? Welke soorten RoleNames zijn er? Gevonden personen hieraan toevoegen of eigen property?
```

Comment:
een roleName kan een URL of Text  zijn. maw, je hoeft dit nu niet vast te leggen. We hebben natuurlijk wel al een lijst hiervan en die gaan we wellicht op termijn in een thesaurus gieten, een skos:Concept dus.